src/pycontw2016/settings/base.py

```diff
@@ -323,8 +323,8 @@ def node_bin(name):
 )
 
 EVENTS_DAY_NAMES = collections.OrderedDict([
-    (datetime.date(2020, 9, 5), _('Day 1')),
```

Comment:
already done in #1030. may need to rebase your branch to the latest master

Reply:
Fixed